resources/terminology.md

```diff
@@ -0,0 +1,75 @@
+| **Sale**    | A [one-phase][one-phase-url] transaction where the amount is settled when the transaction has succeded. Payment instruments using sales transactions are Swish and Direct Bank Debit.                                                                                                                                                                                                                                                                                                                                                                                               |
+| **Two-phase payment flow**            | A payment done in [two steps] [two-phase-payments]. Authorization and capture. The most common payment instrument using two-phase payments is card payments.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| **Verify**     | The payment operation that initiates a [verification][verify-url] payment process. It is a payment that lets you post verifications to confirm the validity of card information, without reserving or charging any amount. This option is used to generate a payment- or recurrence token, that can be used in a recurring payments scenarios or for one-clickpayments, without charging the card in the process. |
+
+[admin-manual]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin/
+[admin-portal]: https://admin.externalintegration.payex.com/psp/login
+[admin-getting-started]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin/
+[admin-interface-search]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin-account-interface-and-search/
+[auto-capture]: /payments/card/other-features#capture
+[hateoas]: https://en.wikipedia.org/wiki/HATEOAS
+[invoice-url]: /payments/invoice/
+[two-phase-payments]: /payments/#the-fundamentals
+[callback-url]:/payments/card/other-features#callback
+[cancel-url]: /checkout/after-payment#cancel
```

Comment:
Why is this the only link that goes to `/checkout` when the other links to specific pages go to `/payments/card`?

Reply:
changed